### PR TITLE
Color pytest logs

### PR DESCRIPTION
**Pull request by harupy**
Signed-off-by: harupy <17039389+harupy@users.noreply.github.com>

## Motivation

Color pytest logs to make it easier to find failures and warnings

## Description of the changes

Add a new entry to `pyproject.toml`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,3 +27,6 @@ lines_after_imports = 2
 force_single_line = 'True'
 force_sort_within_sections = 'True'
 order_by_type = 'False'
+
+[tool.pytest.ini_options]
+addopts = "--color=yes"
```
Comment (harupy) [inline pyproject.toml:32]: Reference: https://docs.pytest.org/en/stable/customize.html#pyproject-toml

Comment (harupy) [reply pyproject.toml:31]: Screen Shot 2021-02-27 at 11 29 13

**Review by crcrpar — APPROVED** [review]

**Comment by harupy** [comment]
@crcrpar Thanks!

**Review by HideakiImamura — APPROVED** [review]
LGTM!

**Pull request merged**